index.html

```diff
@@ -115,6 +115,32 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
One should write meaningful comment so it would be great if you can change your comment to ` * Nick * Contributor card START`.